spec/system/admin/order_spec.rb

```diff
@@ -966,13 +966,18 @@ def new_order_with_distribution(distributor, order_cycle)
               click_link 'Ship Order'
 
               within ".reveal-modal" do
-                expect(page).to have_checked_field('Send a shipment/pick up ' \
-                                                   'notification email to the customer.')
+                expect(page).to have_checked_field(
+                  'Send a shipment/pick up notification email to the customer.'
+                )
                 expect {
                   find_button("Confirm").click
+
+                  # the best & easiest way to close the modal without calling all the JS
+                  find_button("Cancel").click
```

Comment:
This seems to be a bug that needs to be resolved. Testing locally, the modal didn't close for me in dev environment either.

Reply:
I deleted this part to work with the shared examples. In the new version, it closes well after running examples dozens of time. Hope it will be the same in your local and dev.